src/platform/datapath_winuser.c

```diff
@@ -159,6 +159,7 @@ typedef struct DATAPATH_RX_IO_BLOCK {
         WSA_CMSG_SPACE(sizeof(IN6_PKTINFO)) +   // IP_PKTINFO
         WSA_CMSG_SPACE(sizeof(DWORD)) +         // UDP_COALESCED_INFO
         WSA_CMSG_SPACE(sizeof(INT)) +           // IP_ECN
+        WSA_CMSG_SPACE(sizeof(INT)) +           // IP_TOS
```

Comment:
Why do you keep the space for ECN?

Reply:
experiment to try to see if the recv is failing is because there's not enough space. It's not the reason.